client/pages/dapp.js

```diff
@@ -18,19 +19,18 @@ class Dapp extends React.Component {
     this.setState({ balance: response.toNumber() })
   }
 
-  render() {
-    const { web3, accounts, contract } = this.props
```

Comment:
I actually want to leave this one in here, because I want to make it clear that all three props are automatically injected. But we probably should actually use these objects in some manner.

Any ideas?